src/bun.js/webcore/response.zig

```diff
@@ -626,6 +628,9 @@ pub const Fetch = struct {
         signal: ?*JSC.AbortSignal = null,
         aborted: std.atomic.Atomic(bool) = std.atomic.Atomic(bool).init(false),
 
+        //needed to shutdown when aborted
+        socket: ?*uws.Socket = null,
```

Comment:
do we still need this?

Reply:
i forgot this unused, removed on https://github.com/oven-sh/bun/pull/2143/commits/77368b63a8bca0fa27d529556d156198aea87a38